config/initializers/rollbar.rb

```diff
@@ -0,0 +1,53 @@
+Rollbar.configure do |config|
+  config.access_token = Rails.application.secrets.rollbar_token
+
+  # Without configuration, Rollbar is enabled in all environments.
+  # To disable in specific environments, set config.enabled=false.
+  config.enabled = Rails.env.production? && config.access_token.present?
+
+  # By default, Rollbar will try to call the `current_user` controller method
+  # to fetch the logged-in user object, and then call that object's `id`,
+  # `username`, and `email` methods to fetch those properties. To customize:
+  config.person_method = nil
```

Comment:
I'd do something like  

```ruby
def id_as_hash
   Digest::Sha2.hexdigest("#{some_secret_from_deployment_confg}_#{id})
end
```

The purpose of the secret is to act as a bit of salt, otherwise you can precompute a table by feeding incremental integers into the hash algorithm really easily and infer things about our database without access to them. It's paranoid but makes reversing more trouble than it's really worth.